### DISCUSSION TITLE
Just wanted to know... DB schema

### DISCUSSION THREAD

**Discussion by vitalijalbu** [craftcms/cms#12414] [General]
Just wanted to know... why craftcms doesn't adopt (or copy better saying) strapi or directus schema of entries?
Meaniing, in craft we have only one table in db called **entries** but on strapi or directus every collection (entry type, channel or whatever) have separate tables.
This will affect response time of course if having a large amount of entries.
@brandonkelly just curious 🥇

**Comment by brandonkelly** [comment]
We considered creating a table for each section but it was a net negative for performance since you can query entries across multiple sections at once. And entries are differentiated by entry type, not section.
